pages/about/community.vue

```diff
@@ -1,29 +1,60 @@
 <template>
     <i18n-page-wrapper>
-        <core-h1 :title="$t('title')"></core-h1>
-        <div
-            class="h-64 md:h-96 lg:h-96 w-full mt-4 mb-8"
-            :style="communityMapStyle"
-        ></div>
-        <div class="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-3 gap-8">
-            <div
-                v-for="community in $t('communities')"
-                :key="`community_${community.tag}`"
-            >
-                <div class="flex flex-col box p-5 min-h-full">
-                    <ext-link
-                        :href="communityLink[community.tag]"
-                        class="community-title"
-                    >
-                        {{ community.title }}
-                    </ext-link>
-                    <p
-                        v-for="(paragraph, i) in community.desc"
-                        :key="`paragraph_${i}`"
-                        class="text-xs"
+        <core-h1 :title="$t('title')" class="page-title"></core-h1>
+        <div class="flex flex-col justify-between lg:flex-row">
+            <div class="map-area">
+                <p v-text="$t('content')"></p>
+                <div class="map" :style="communityMapStyle"></div>
+            </div>
+            <div class="communities">
+                <div class="arrow-top" :class="{ hide: boxScroll }">⌃</div>
+                <div ref="communities" class="content">
+                    <div
+                        v-for="(community, index) in $t('communities')"
+                        :key="`community_${community.tag}`"
                     >
-                        {{ paragraph }}
-                    </p>
+                        <div class="box" @click="popupShow(index)">
+                            <div class="text">
+                                <h3 class="title">
+                                    {{ community.title }}
+                                </h3>
+                                <p>
+                                    {{ community.description }}
+                                </p>
+                            </div>
+                            <img :src="community.imgUrl" :alt="community.tag" />
```

Comment:
How about adding `v-show` to display the image only when the image is existent?

Reply:
I got it.